lib/react.js

```diff
@@ -239,6 +239,22 @@ declare module 'react-dom' {
   ): React$Component<$DefaultPropsOf<Config>, $PropsOf<Config>, any>;
 
   declare function unmountComponentAtNode(container: any): boolean;
+
+  declare function unstable_batchedUpdates(
+    callback: Function,
+    a?: any,
```

Comment:
I was skeptical that the args were named like this. [I was wrong](https://github.com/facebook/react/blob/62c9d999bef942383215f8dcaaf65fd06160eb60/src/renderers/shared/reconciler/ReactUpdates.js#L110) :stuck_out_tongue: